skll/learner/__init__.py

```diff
@@ -1618,7 +1618,8 @@ def learning_curve(self,
                        examples,
                        metric,
                        cv_folds=10,
-                       train_sizes=np.linspace(0.1, 1.0, 5)):
+                       train_sizes=np.linspace(0.1, 1.0, 5),
+                       min_training_examples=500):
```

Comment:
Hmm, I am not sure we want to make this an argument? If we know that less than 500 examples lead to unreliable curves, why not simply hardcode that and add an argument like `override_minimum` which is `False` by default but users can choose to override it if they really want to. So, in this case, we would raise a `ValueError` if the number of features doesn't meet the 500 minimum. We would downgrade that error to a warning though if the user sets `override_minimum` to `True`. 

What do you guys think @aoifecahill @mulhod @bndgyawali ?

Reply:
BTW, whatever we end up doing, we would need to add new tests.

Reply:
Yeah, `override_minimum` sounds good to me.